### PR TITLE
Mask according to the paper

### PR DESCRIPTION
**Pull request by stevenhillis**
In section _3.2. Masking_ of the SoundStorm paper, they describe the masking strategy they use during training. This strategy was not implemented in the repo; the mask was just random: 
before
I could not overfit even a toy dataset with the random masking. After making these changes to align with the paper, I successfully overfit toy data and am seeing good signs with a larger scale train.
after

### CODE REVIEW

**Comment by lucidrains** [comment]
@stevenhillis yes, you are correct! i missed this section

and your code is great! thank you so much!
